app.js

```diff
@@ -297,6 +415,18 @@ app.post('/hubs/:hubSlug/start_activity', function(req, res){
   }
 })
 
+app.post('/hubs/:hubSlug/devices/:deviceSlug/activate_command', function(req, res){
```

Comment:
So you think we should ditch the larger json object stuff, and just work in slugs? This is related to the comment I posted on the main pull, we should probably discuss that there.